_less/screen.less

```diff
@@ -907,9 +907,11 @@ table td,table th{
 	border:2px solid #ee9209;
 	color:#fff;
 }
-#glossary_term {
+.glossary_term {
 	display: block;
 	width: 20em;
+	text-align: center;
```

Comment:
This was made on purpose due to the searchbar being centered and trapped between centered texts, but sure, if you find it weird, please feel free to drop it. If you'd ask me, a centered searchbox seemed usual to me but now that you're mentionning it, I can't find a good example at hands :)